cartridge/graphql/execute.lua

```diff
@@ -189,6 +189,10 @@ local function completeValue(fieldType, result, subSelections, context, opts)
     return completedResult
   end
 
+  if fieldTypeName == 'Scalar' or fieldTypeName == 'Enum' then
+    return fieldType.serialize(result)
+  end
+
```

Comment:
For example, I want to make custom type, that parse and serialize JSON objects. When I use it, I'm parsing response by `json.decode` (if it's Python code) function. If there is no `nil` serialization, then the response will contain a `nil`, but not a json string `'null'`. So, as a result, there will be a parsing error. To prevent it, I will have to use some conditions to handle `nil` separately, it's not uncomfortable. It is also logical that if we want to receive JSON at the output, then it should always be JSON (except for the case with errors).